draft-ietf-masque-h3-datagram.md

```diff
@@ -113,8 +119,8 @@ error of type H3_DATAGRAM_ERROR.
 
 HTTP Datagram Payload:
 
-: The payload of the datagram, whose semantics are defined by individual
-applications. Note that this field can be empty.
+: The payload of the datagram, whose semantics are defined by the
+application. Note that this field can be empty.
```

Comment:
In the abstract, you propose the addition of the text "Both are intended for use by HTTP extensions, not applications." do you think we need to, similarly, avoid using the term application here?

Reply:
Possibly -- the semantics are definitely _contextualised_ by the extension in use, and it's probably up to it to determine whether they're delegated to the application.

how about

```suggestion
extension that is using HTTP Datagrams. Note that this field can be empty.
```